test/integration/configuration/resolveConfig-test.js

```diff
@@ -27,9 +31,57 @@ describe('resolveConfig()', () => {
     it('contains default options', () => {
       assert.hasAllKeys(config, Object.keys(_utils.DEFAULT_CONFIG).concat('emitter'));
     });
+
     it('overrides default options with custom ones', () => {
       assert.deepEqual(config.path, ['./foo.apib']);
     });
+
+    describe('replaces "server" with "endpoint"', () => {
+      it('removes "server" key', () => {
+        assert.notProperty(config, 'server');
+      });
+
+      it('sets value to "endpoint" key', () => {
+        assert.propertyVal(config, 'endpoint', 'http://127.0.0.1');
+      });
+
+      it('allows to set "options.server" command', () => {
```

Comment:
This `it` (and the subsequent ones) feels like it should rather be a `describe`, which has certain config as an input and asserts for both `server` and `endpoint`, so the expected behavior is clearer:

- server = abc ➡️ endpoint = abc, server = default
- server = abc, options.server = xyz ➡️ endpoint = abc, server = xyz
- options.server = abc ➡️ endpoint = default, server = abc
- endpoint = abc ➡️ endpoint = abc, server = default
- endpoint = abc, options.server = xyz ➡️ endpoint = abc, server = xyz
- endpoint = abc, server = xyz ➡️ endpoint = ???, server = default (??? in reality should never happen, might make sense to error and abort)

Reply:
Broken down test suits for `resolveConfig` 74091dcc3763bb5a990a71d5017754c2280db1cb